tests/callbacks/test_image_visualizer.py

```diff
@@ -1,112 +1,235 @@
 # Copyright 2022 MosaicML Composer authors
 # SPDX-License-Identifier: Apache-2.0
 
+import imghdr
+import json
 import os
+import zipfile
 from pathlib import Path
 
 import pytest
 from torch.utils.data import DataLoader
 
 from composer.callbacks import ImageVisualizer
-from composer.core import Time, TimeUnit
-from composer.loggers import InMemoryLogger, WandBLogger
+from composer.loggers import WandBLogger
+from composer.loggers.logger import Logger
 from composer.trainer import Trainer
-from tests.common.datasets import RandomImageDataset
-from tests.common.models import SimpleConvModel
-
-try:
-    import wandb
-    _WANDB_INSTALLED = True
-    del wandb  # unused
-except ImportError:
-    _WANDB_INSTALLED = False
-
-
-@pytest.mark.skipif(not _WANDB_INSTALLED, reason='Wandb is optional')
-@pytest.mark.parametrize('interval', ['9ba', '90ba', '2ep', '3ep', '7ep'])
```

Comment:
Can you add the interval parametrization back to at least one of the tests? There was a bug related to this, so I don't want to regress coverage for that.